albumentations/augmentations/keypoints_utils.py

```diff
@@ -1,5 +1,98 @@
 from __future__ import division
 import math
+import warnings
+
+from albumentations.core.utils import Params, DataProcessor
+
+__all__ = ['check_keypoints', 'convert_keypoints_from_albumentations', 'convert_keypoints_to_albumentations',
+           'filter_keypoints', 'KeypointParams']
+
+keypoint_formats = {'xy', 'yx', 'xya', 'xys', 'xyas', 'xysa'}
```

Comment:
Could you please add a comment that describes what x, y, s, a are?

Or/and add a page to the documentation that has this info.

Reply:
@BloodAxe we need you

Reply:
x - X coordinate,
y - Y coordinate
s - Keypoint scale
a - Keypoint orientation in radians or degrees (depending on KeypointParams.angle_in_degrees)